code/modules/tools/tool_extension.dm

```diff
@@ -51,10 +51,14 @@
 
 	user.visible_message(SPAN_NOTICE("\The [user] begins [start_message || tool_archetype.use_message] \the [target] with \the [holder]."), SPAN_NOTICE("You begin [start_message || tool_archetype.use_message] \the [target] with \the [holder]."))
 	var/use_sound = LAZYACCESS(tool_use_sounds, archetype)
-	if(islist(use_sound) && length(use_sound))
-		use_sound = pick(use_sound)
-	if(use_sound)
-		playsound(user.loc, use_sound, 100)
```

Comment:
You removed the first `playsound()` without replacing it, ftr.

Reply:
I removed it because I wasn't sure why there were 2 play sound really?
Most common tool interactions only play their sound once. So I wasn't sure if that was part of some merge artifact or something? I can add it back if you want.

Reply:
It's a sound for starting and a sound for ending. Most tool interactions in current code don't have a do_after(). Readding it would be ideal.